src/test/java/core/basesyntax/service/CsvFileReaderServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.service.Util.INPUT_FILE_LINES;
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvFileReaderServiceImplTest {
+    private static FileReaderService readerService;
+    private static final String PATH_TO_FILE = "src/test/resources/inputFile.csv";
+
+    @BeforeClass
+    public static void beforeClass() {
+        readerService = new CsvFileReaderServiceImpl();
+    }
+
+    @After
+    public void tearDown() {
+        new File(PATH_TO_FILE).delete();
+    }
+
+    @Test
+    public void readFromFile_defaultCase_Ok() {
+        writeToFileText();
+        List<String> expected = INPUT_FILE_LINES;
+        List<String> actual = readerService.readFromFile(PATH_TO_FILE);
+        assertEquals("Expected should be equal to "
+                + expected + " but was: "
+                + actual, expected, actual);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void readFromFile_fileDoesNotExist_notOk() {
+        readerService.readFromFile(PATH_TO_FILE);
+    }
+
+    public static void writeToFileText() {
+        Util.createNewFolder();
```

Comment:
Lets move this `Util.createNewFolder();` to `readFromFile_defaultCase_Ok()` method